assets/sass/components/overlay-notification/_googlesitekit-overlay-notification.scss

```diff
@@ -43,6 +43,12 @@
 		}
 	}
 
+	@media (min-width: $page-content-max-width) {
```

Comment:
Arguably this rule should kick in from `min-width: $page-content-max-width + 1`, but in practical terms it won't make a difference to leave as-is...